packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts

```diff
@@ -275,3 +262,87 @@ describe('listAccountHistory', () => {
     expect(history.find((h) => h.type === 'AccountToUtxos' && h.amounts[0] === '-3.97000000@DFI')).toBeTruthy()
   })
 })
+
+describe('listAccountHistory for poolpair', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+  const createToken = createTokenForContainer(container)
+
+  let address: string
```

Comment:
we don't use global variable as new practice now for test independence
you might need to move `address` in each test

Reply:
I have made changes to this.